gapil/template/constant_sets.go

```diff
@@ -188,6 +188,25 @@ func buildConstantSets(api *semantic.API, mappings *resolver.Mappings) *Constant
 			constsets[p] = b.addLabels(ev.Labels, ev.Ty.IsBitfield)
 		}
 	}
+	for _, e := range api.Enums {
+		unique := false
+		for _, annotation := range e.Annotations {
+			if annotation.Named == semantic.Named("unique") {
+				unique = true
+				break
+			}
+		}
+		if unique {
+			labels := analysis.Labels{}
+			for _, entry := range e.Entries {
+				// Assume the enums are uint32
```

Comment:
Enums only default to U32, they can be specified to be anything.

`Enum.NumberType` specifies the type of the enum. If we want to only support 32-bit for now, that's fine but we should error out of we can't handle it, rather than silently continue.